pytensor/link/pytorch/dispatch/extra_ops.py

```diff
@@ -21,3 +21,44 @@ def cumop(x):
             return torch.cumprod(x, dim=dim)
 
     return cumop
+
+
+@pytorch_funcify.register(Repeat)
+def pytorch_funcify_Repeat(op, **kwargs):
+    axis = op.axis
+
+    def repeat(x, repeats, axis=axis):
+        return x.repeat_interleave(repeats, dim=axis)
+
+    return repeat
+
+
+@pytorch_funcify.register(Unique)
+def pytorch_funcify_Unique(op, **kwargs):
+    return_index = op.return_index
+
+    if return_index:
+        # TODO: evaluate whether is worth implementing this param
+        # (see https://github.com/pytorch/pytorch/issues/36748)
+        raise NotImplementedError("return_index is not implemented for pytorch")
+
+    axis = op.axis
+    return_inverse = op.return_inverse
+    return_counts = op.return_counts
+
+    def unique(
+        x,
+        return_index=return_index,
+        return_inverse=return_inverse,
+        return_counts=return_counts,
+        axis=axis,
```

Comment:
We don't need these optional kwargs. I know jax implementations were doing it but I don't see why. These functions are never called with them changed.

Its enough to rely on the scope to access them